src/test/java/ua/edu/internship/interview/service/business/UserSkillsServiceTest.java

```diff
@@ -0,0 +1,180 @@
+package ua.edu.internship.interview.service.business;
+
+import org.bson.types.ObjectId;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.extension.ExtendWith;
+import org.mockito.InjectMocks;
+import org.mockito.Mock;
+import org.mockito.junit.jupiter.MockitoExtension;
+import ua.edu.internship.interview.data.documents.SkillDocument;
+import ua.edu.internship.interview.data.documents.UserSkillsDocument;
+import ua.edu.internship.interview.data.repository.SkillRepository;
+import ua.edu.internship.interview.data.repository.UserSkillRepository;
+import ua.edu.internship.interview.service.dto.skill.SkillDto;
+import ua.edu.internship.interview.service.dto.user.skill.UserSkillsDto;
+import ua.edu.internship.interview.service.mapper.UserSkillsMapper;
+import ua.edu.internship.interview.service.utils.exceptions.InvalidInputException;
+import ua.edu.internship.interview.service.utils.exceptions.NoSuchEntityException;
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertNotNull;
+import static org.junit.jupiter.api.Assertions.assertThrows;
+import static org.mockito.ArgumentMatchers.any;
+import static org.mockito.Mockito.verify;
+import static org.mockito.Mockito.when;
+import java.util.List;
+import java.util.Optional;
+
+@ExtendWith(MockitoExtension.class)
+class UserSkillsServiceTest {
+    @Mock
+    private UserSkillRepository userSkillRepository;
+    @Mock
+    private SkillRepository skillRepository;
+    @Mock
+    private UserSkillsMapper userSkillsMapper;
+    @InjectMocks
+    private UserSkillsService underTest;
+
+    private List<SkillDocument> skillDocuments;
+    private List<SkillDto> skillDTOs;
```

Comment:
🚫 **[checkstyle]** <com.puppycrawl.tools.checkstyle.checks.naming.AbbreviationAsWordInNameCheck> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>Abbreviation in name 'skillDTOs' must contain no more than '2' consecutive capital letters.
<!-- __reviewdog__:ChBiMmE1MzQyOWI1YjUxZmNmEgpjaGVja3N0eWxl -->